common/goal_distance_calculator/config/goal_distance_calculator.param.yaml

```diff
@@ -1,3 +1,4 @@
 /**:
   ros__parameters:
     update_rate: 10.0
+    oneshot: true
```

Comment:
@PhoebeWu21  oneshot should not be changed at this PR.

Reply:
@taikitanaka3 
According to the new [parameter coding guidelines](https://autowarefoundation.github.io/autoware-documentation/main/contributing/coding-guidelines/ros-nodes/parameters/), any `declare_parameter` function should have a parameter in the parameter file. Why should the `oneshot` parameter not exists in the parameter file?

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/4830/files#r1329414651

or please do not add delete change to launcher files.

Reply:
@taikitanaka3 Got it! Done.